apps/cowswap-frontend/src/modules/sounds/utils/sound.ts

```diff
@@ -47,7 +45,12 @@ function isDarkMode(): boolean {
 }
 
 function getThemeBasedSound(type: SoundType): string {
-  const featureFlags = jotaiStore.get(featureFlagsAtom) as Record<string, boolean>
+  // TODO: load featureFlags when enabling again
```

Comment:
nit: leave it like this, but here not sure you should leave a TODO. We really won't do anything here. We will just come back when we have another seasonal theme, but this TODO is something we don't want to act right now